### ISSUE TITLE
[Mod incompatibility] Blur

### ISSUE THREAD

**Issue by Sythiex** [jrbudda/Vivecraft_112#108]
In-game menus (inventory, etc) whitescreen and are not usable.

Forge 14.23.5.2838
Vivecraft 1.12.2 v7
Blur 1.0.4

[https://minecraft.curseforge.com/projects/blur](https://minecraft.curseforge.com/projects/blur)

**Comment by Techjar** [comment]
Yeah this is known, just remove the mod, there's no downside and it'll never be compatible.
